python/pyspark/sql/dataframe.py

```diff
@@ -1422,6 +1422,53 @@ def colRegex(self, colName: str) -> Column:
         jc = self._jdf.colRegex(colName)
         return Column(jc)
 
+    def convert(self, schema: StructType) -> "DataFrame":
+        """
+        Returns a new DataFrame where each row is reconciled to match the specified schema.
+        Spark will:
```

Comment:
IIRC, it should have one newline to make it pretty in the API reference. Does it render fine without the newline??